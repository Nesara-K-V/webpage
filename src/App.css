body {
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; /* Specify the font family (Arial) and a generic font family (sans-serif) as a fallback */
  font-size: 16px; /* Set the font size to 16 pixels */
  color: #333; /* Set the text color to a shade of gray (#333) */
}
.header{
  
  display: flex;
  margin-top: 1%;
  height: 5%;
  
}
.heading{
  margin-left: 20%;
  
}
.heading  h3{
  margin:3% 0%;
  color:rgb(104, 97, 97);
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
  
}
.heading  h2{
  margin: 5% 3%;
}

.menu_items{
  display: flex;
  list-style: none;
  }

.menu{
  margin-left: 30%;
}

.menu li{
  padding:10% 10%;
}
.menu li:hover{
  background-color: #333;
  color: aliceblue;
}
.bg{
  width:100%;
  height: 100%;
  margin-top: 20px;
  
}
.opening_page{
  height: 500px;
  width: 100%;
}
.text-overlay {
  position: absolute;
  top: 40%;
  left: 50%;
  transform: translate(-50%, -50%);
  color: white;
  font-size: 90px;
}
.text-overlay1 {
  
  position: absolute;
  top: 52%;
  left: 50%;
  transform: translate(-50%, -50%);
  color: white;
  font-size: 90px;
}
@media screen and (min-width: 769px) and (max-width: 1024px) {
  .header{
  display: block;
  width:100%;
  }
  .opening_page{
    width: 100%;
  }
  .header h2{
    margin-bottom: 0%;
  }
  .heading{
    left:50%;
  }
  .menu_items{
    margin-left: 0%;
    display: flex;
    list-style: none;
  }
  .menu{
    width:80%;
    margin-left: 5%;
  }
    
  .menu li{
      padding:2% 2%;
      margin: 0% 3%;
  }
  .bg{
    width:100%;
    margin-top:1px;
  }
  .text-overlay {
    position: absolute;
    top: 33%;
    left: 35%;
    transform: translate(-50%, -50%);
    color: white;
    font-size: 400%;
  }
  .text-overlay1 {
    
    position: absolute;
    top: 38%;
    left: 35%;
    transform: translate(-50%, -50%);
    color: white;
    font-size: 400%;
  }

}

