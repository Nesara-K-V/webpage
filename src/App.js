import logo from './logo.svg';
import './App.css';
import { Header_bar } from './mainpage/Header_bar';
import { Main_p } from './mainpage/Main_p';

function App() {
  return (

    <>
    <h1>NESARA</h1>
    <Header_bar />
    <Main_p /></>
    
  );
}

export default App;
