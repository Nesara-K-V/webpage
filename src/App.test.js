import { render, screen } from '@testing-library/react';
import App from './App';

test('App component renders without errors', () => {
  render(<App />);
});
test('App renders Header_bar and Main_p components', () => {
  const { getByText } = render(<App />);
  
  // Test that specific text content from Header_bar and Main_p components is present
  expect(getByText('CAROL SOLVAY')).toBeInTheDocument();
  expect(getByText('HIGHEST')).toBeInTheDocument();
  expect(getByText('POTENTIAL')).toBeInTheDocument();
  expect(getByText('HOME')).toBeInTheDocument();
  expect(getByText('ABOUT')).toBeInTheDocument();
  expect(getByText('CONTACT')).toBeInTheDocument();

});
test('App matches snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
