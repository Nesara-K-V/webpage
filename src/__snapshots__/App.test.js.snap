// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="heading"
    >
      <h2>
        CAROL SOLVAY
      </h2>
      <h3>
        Mindset Mastery Mentor
      </h3>
    </div>
    <div
      class="menu"
    >
      <ul
        class="menu_items"
      >
        <li>
          HOME
        </li>
        <li>
          ABOUT
        </li>
        <li>
          CONTACT
        </li>
      </ul>
    </div>
  </div>
  <div
    class="opening_page"
  >
    <img
      alt="An example image"
      class="bg"
      src="bg.jpg"
    />
    <div
      class="text-overlay"
    >
      <h1>
        HIGHEST
      </h1>
      <h1>
        POTENTIAL
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
